util/metrics/src/lib.rs

```diff
@@ -48,6 +48,8 @@ pub struct Metrics {
     pub ckb_freezer_read: IntCounter,
     /// Counter for relay transaction short id collide
     pub ckb_relay_transaction_short_id_collide: IntCounter,
+    /// Histogram for relay compact block verify time
+    pub ckb_relay_cb_verify_time: Histogram,
```

Comment:
How about `ckb_relay_cb_verify_timecost`?

Reply:
renamed to `ckb_relay_cb_verify_duration`